### PR TITLE
Add docs template

### PR DESCRIPTION
**Pull request by blankdots**
### Description

Add documentation using sphinx in order to prepare for read the docs.

Built version: https://metadata-submitter.readthedocs.io/en/feature-add-docs/

### Related issues

Fixes (partly) #26 

### Type of change

- [x] New feature (non-breaking change which adds functionality)

### Changes Made

1. Add template documentation using sphinx
2. add `.docs` to setup install
3. add docs tests in tox.ini

### Testing

- [x] Unit Tests

### CODE REVIEW

File: docs/make.bat [hunk 1]
```diff
@@ -0,0 +1,35 @@
+@ECHO OFF
```
Comment (otahontas) [inline docs/make.bat:1]: Why do we have this bat-file here?

Comment (blankdots) [reply docs/make.bat:1]: auto generated using `sphinx-quickstart` I can remove it, no big deal

Comment (otahontas) [reply docs/make.bat:1]: Not a problem, was just thinking if its important. We can leave it

**Comment by otahontas** [comment]
@blankdots is this ok to merge?

**Comment by blankdots** [comment]
@otahontas go ahead.

**Pull request merged**